dnsmasq_interface.c

```diff
@@ -410,29 +384,14 @@ void FTL_reply(unsigned short flags, char *name, struct all_addr *addr, unsigned
 		{
 
 			// Save reply type and update individual reply counters
-			save_reply_type(flags, i);
-
-			// Save returned IP
-			// but only if we have an exact match (there will be no exact match for a CNAME)
-			// on this else-branch this is not a negative match (NXDOMAIN, NODATA)
-			if(!(flags & F_NEG) && !(flags & F_CNAME) &&
-			   strlen(dest) > 2)
-			{
-				storeIP(i, dest);
-			}
-
-			// Store TTL
-			queries[i].ttl = ttl;
-
-			// Save response time (relative time)
-			queries[i].response = response.tv_sec*10000 + response.tv_usec/100 - queries[i].response;
+			save_reply_type(flags, i, response);
 		}
 	}
 	else if((flags & F_REVERSE) && debug)
 	{
 		logg("Skipping result of PTR query");
 	}
-	else if(debug)
+	else
```

Comment:
Yes, during the beta testing phase, I think this should go into the logs of the users so that they could tell us: Hey guys, FTL told me you you missed exotic query type XYZ.